FacilityDB/3 reports.sql

```diff
@@ -0,0 +1,31 @@
+--AS: you don't have proper sample code for most of your questions, just letting you know.
+--The total amount of Telehealth appts. comparing the last three years.
```

Comment:
It is showing the count of all appts., wanted only the Tele-visits.

Reply:
Sorry missed that. done.